control/modelsimp.py

```diff
@@ -219,12 +219,12 @@ def balred(sys, orders, method='truncate', alpha=None):
     method: string
         Method of removing states, either ``'truncate'`` or ``'matchdc'``.
     alpha: float
-        Specifies the alpha-stability boundary for the eigenvalues
-        of the state dynamics matrix A. For a continuous-time
-        system, alpha <= 0 is the boundary value for the real parts
-        of eigenvalues, while for a discrete-time system, 0 <= alpha <= 1
-        represents the boundary value for the moduli of eigenvalues.
-        The alpha-stability domain does not include the boundary.
+        Redefines the stability boundary for eigenvalues of the system matrix A.
+        By default for continuous-time systems, alpha <= 0 defines the stability
+        boundary for the real part of A's eigenvalues and for discrete-time
+        systems, 0 <= alpha <= 1 defines the stability boundary for the modulus
+        of A's eigenvalues.See SLICOT routines AB09MD and AB09ND for more
```

Comment:
While resolving the merge conflict, I added a space here after the `.`. So, now it reads "...eigenvalues. See SLICOT...".